pathfinding_service.rst

```diff
@@ -267,11 +271,11 @@ Example
     }
     // Result for failure
     {
-        "error": "No suitable path found."
+        "errors": "No suitable path found."
```

Comment:
Should it be possible to have multiple errors here, e.g. `The token network address is invalid` and `Bias is invalid`, or just the first one?

Reply:
From what I've seen, the raiden client only returns a single error. If we change that, we should do it consistently across all raiden repos.

Reply:
yes I think the idea in raiden was that we would in the future have multiple errors. So that this could either be a list or string.